torch/lib/THC/THCNumerics.cuh

```diff
@@ -44,6 +44,7 @@ struct THCNumerics<char> {
   static inline __host__ __device__ bool eq(char a, char b) { return a == b; }
   static inline __host__ __device__ bool ne(char a, char b) { return a != b; }
 
+  static inline __host__ __device__  char neg(char a) { return -a; }
```

Comment:
It's not needed for the feature, but it seems reasonable to be able to call neg on a char in the library.